301-sf-preview-windows-containers/azuredeploy.json

```diff
@@ -0,0 +1,565 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "clusterName": {
+          "type": "string",
+          "defaultValue": "sfcluster",
+          "metadata": {
+              "description": "Name of your cluster - Between 3 and 23 characters. Letters and numbers only"
+          }
+      },
+      "adminUserName": {
+          "type": "string",
+          "metadata": {
+              "description": "Remote desktop user Id"
+          }
+      },
+      "adminPassword": {
+          "type": "securestring",
+          "metadata": {
+              "description": "Remote desktop user password. Must be a strong password"
+          }
+      },
+      "vmImagePublisher": {
```

Comment:
move all the VM image params to variables or just use literal values if used only once